### PR TITLE
added all of the keys and octaves that would appear on standard piano…

### PR DESCRIPTION
**Pull request by sarastavnitser**
… keyboard

### CODE REVIEW

File: src/main/java/piano/keyboard/keyboardaudio/Notes.java [hunk 1]
```diff
@@ -4,12 +4,12 @@
 
 public interface Notes
 {
-	int C = 60;
+	int C =57;
```
Comment (andrewoid) [inline src/main/java/piano/keyboard/keyboardaudio/Notes.java:7]: These values were wrong?

File: src/main/java/piano/keyboard/keyboardui/Keyboard.java [hunk 2]
```diff
@@ -29,8 +29,16 @@ public Keyboard(MidiChannel midiChannel, Recorder recorder) {
...
-
         int index = 0;
+
+        addPianoLabel(Color.WHITE, index, placement);
+        placement += KeyStats.WHITE_WIDTH + KeyStats.SPACE_BETWEEN_WHITE_KEYS;
+        index += 2;
+
+        addPianoLabel(Color.WHITE, index, placement);
+        placement += KeyStats.WHITE_WIDTH + KeyStats.SPACE_BETWEEN_WHITE_KEYS;
+        index++;
```
Comment (andrewoid) [inline src/main/java/piano/keyboard/keyboardui/Keyboard.java:40]: Please include a screenshot of the new UI in your PR.

Drag an image to the PR description to upload it.

File: src/main/java/piano/keyboard/keyboardaudio/Notes.java (hunk 1 above)
Comment (sarastavnitser) [reply src/main/java/piano/keyboard/keyboardaudio/Notes.java:7]: I added three keys to the bottom of the piano that are before C so I updated the note numbers and shifted them by three

**Comment by sarastavnitser** [comment]
Screen Shot 2022-09-24 at 9 32 53 PM

**Pull request merged**